src/java.base/windows/native/libjli/java_md.c

```diff
@@ -606,56 +606,69 @@ JLI_ReportErrorMessage(const char* fmt, ...) {
 }
 
 /*
- * Just like JLI_ReportErrorMessage, except that it concatenates the system
- * error message if any, it's up to the calling routine to correctly
- * format the separation of the messages.
+ * Because Windows API errors and regular errno errors can exist at the same time,
+ * compromise and error out when both are present. Windows code should ideally clear
+ * the last error before a call that sets either of these!
```

Comment:
That comment is strange. Of course Windows resets both errno or GetLastError if an API using them is called. But these mechanisms have nothing to do with each other and resetting errno if a WIN32 call is done would be wrong, and vice versa.